docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+* Create and remove "future associations" (also known as "Pending Enterprise Customer Users").
+
+In addition, during linking student(s) to Enterprise Customer it is capable of enrolling them into Course or Program.
+
+.. image:: images/manage-learners.png
+
+Integrating with Course Catalog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order for course enrollment features to work, `edx-enterprise` talks to `Course Catalog IDA`_. Instructions on
+setting up and configuring Course Catalog can be found at corresponding `docs section` of Course Catalog and in
+`edx-enterprise` `Pull Request #7`_ (describes developer setup, but useful for a broader audience).
+
+One particular quirk is that Course Catalog IDA have internal access management, so administrators should access
+`edx-enterprise` admin interface using account registered with Course Catalog IDA. This is covered abovementioned
+docs section and pull request.
+
+.. _Course Catalog IDA: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html
+.. _docs section: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html#lms-integration
+.. _Pull Request #7: https://github.com/edx/edx-enterprise/pull/7
+
+Managing Linked Learners Lists
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Manage Learners view allows for two modes of linking:
+
+* Singular - single user email or username is provided.
+* Bulk - CSV file containing single column of emails is uploaded.
+
+When linking single user by username, `edx-enterprise` tries to find an existing user with that username and fails the
+linking if match was not found. When email is used (both in singular and bulk modes), existing users using that email
+are linked to Enterprise Customer; if email match was not found a "Pending linked learner" record is created for that
+email. When that email is used to register a new user that user is automatically linked with Enterprise customer.
+
+**Note:** at the moment of writing, each learner can be linked only to one Enterprise Customer, so linking fails if
+learner is already associated with some other Enterprise Customer.
+
+The view also allows unlinking "Linked Learners" and "Pending Linked Learners".
+
+Enrolling Learners into Course or Program
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enrolling learners into course or program depends on Course Catalog integration. In two words, this feature uses
+Course Catalog API to fetch course/program information.
+
+Linked Learner Enrollment only happens as a side-effect of linking user to an Enterprise Customer, and supports both
+"Singular" and "Bulk" modes.
+
+**Note:** At the moment of writing, enrolling into course/program was exclusive - i.e. learner(s) are enrolled either
+into single course, or into all courses within a program.
+
+Enrollment notification email templates
+---------------------------------------
+
+When learners are enrolled into a course/program, admins might opt-in (on per-site basis) to send enrollment
```

Comment:
Currently, whether or not emails get sent is actually something that you opt-in when you submit the form to enroll specific learners; it's not set on a per-enterprisecustomer or per-site basis. (Though the appearance/templates used for those emails can be set on a per-site basis.)